File: app-mvc-java/src/main/java/com/naveed/samples/data/models/ListMoviesResp.java
package com.naveed.samples.data.models;

import java.util.List;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ListMoviesResp {

    @SerializedName("page")
    @Expose
    public Integer page;
    @SerializedName("total_results")
    @Expose
    public Integer totalResults;
    @SerializedName("total_pages")
    @Expose
    public Integer totalPages;
    @SerializedName("results")
    @Expose
    public List<MovieItem> results = null;

}

File: app-mvp/src/main/java/com/naveed/samples/data/models/ListMoviesResp.kt
package com.naveed.samples.data.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ListMoviesResp {

    @SerializedName("page")
    @Expose
    var page: Int? = null
    @SerializedName("total_results")
    @Expose
    var totalResults: Int? = null
    @SerializedName("total_pages")
    @Expose
    var totalPages: Int? = null
    @SerializedName("results")
    @Expose
    var results: List<MovieItem>? = null

}